.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -72,4 +72,4 @@ body:
     id: additional-context
     attributes:
       label: Logs, Errors, Screenshots, and Additional Context
-      description: LLM logs will be stored in the `logs/llm/default` folder. Please add any additional context about the problem here.
+      description: You can share the chat history using the thumbs-down (👎) button above the input field. LLM logs will be stored in the `logs/llm/default` folder. Please add any additional context about the problem here.
```

Comment:
```suggestion
      description: If you want to share the chat history you can click the thumbs-down (👎) button above the input field and you will get a shareable link (or you can also click thumbs up when things are going well of course!). LLM logs will be stored in the `logs/llm/default` folder. Please add any additional context about the problem here.
```

Reply:
That is definitely clearer.
I'd just like to avoid any misunderstanding by the user: the feedback tool is solely about the content of a current chat's history. It is neither desired nor used for any technical troubleshooting within bug reports on OpenDevin's GitHub pages.

Reply:
> (or you can

@neubig Removing "or" makes the sentence more concise and avoids redundancy, as the thumbs-up (👍) option is an additional note rather than an alternative action.

Reply:
Done!